Purchases/Public/RCPurchases.m

```diff
@@ -306,13 +308,9 @@ - (void)getEntitlements:(RCReceiveEntitlementsBlock _Nullable)completion
 
                                            if (entitlements != nil) {
                                                self.cachedEntitlements = entitlements;
-                                           } else {
-                                               self.cachesLastUpdated = nil;
```

Comment:
yeah, so same here, when this function is called from updateCaches is fine because this is updated somewhere else, but when this is called from `(void)entitlementsWithCompletionBlock:(RCReceiveEntitlementsBlock)completion`, we never update this

Reply:
Yeah, which practically translates into there being no expiration for the entitlements cache which might be ok for now?